mautic/mautic
# Allows non-numeric values for radio button

Related to issue #4265

The twitter button size passes in the values 'medium' and 'large' which
get typecast to an interger resulting in two options with a value of 0.

Removing the typecasting creates a button with 'yes' and 'no' labels
with values of 'medium' and 'large'.

To test:

- Go to the plugins page and open the Twitter options.
- Check the features tab - the 'Large button' option will have 2 'yes' labels which are both selected. Both values are 0.
- Make the change in this commit and refresh the page.
- The button should now have 'yes' and 'no' labels with only one option selected. The values should now be 'medium' and 'large'
